engines/ultima/nuvie/metaengine.cpp

```diff
@@ -56,133 +56,133 @@ struct NuvieActionDescription {
+	{ "QUICK_LOAD", _s("Quick load"), nullptr, nullptr },
+	{ "QUIT", _s("Quit"), "q", "C+q" },
+	//{ "QUIT_NO_DIALOG", "_s(Quit without confirmation"), "M+q", nullptr },
+	{ "GAME_MENU_DIALOG", _s("Show game menu; Cancel action if in middle of action"), "ESCAPE", nullptr },
+	//{ "TOGGLE_FULLSCREEN", "_s(Toggle fullscreen"), "M+RETURN", nullptr },
+	{ "TOGGLE_CURSOR", _s("Toggle Cursor"), "TAB", nullptr },
+	{ "TOGGLE_COMBAT_STRATEGY", _s("Toggle combat strategy"), "BACKQUOTE", nullptr },
+	{ "TOGGLE_FPS_DISPLAY", _s("Toggle frames per second display"), "C+PLUS", nullptr },
+	{ "TOGGLE_AUDIO", _s("Toggle audio"), "C+a", nullptr },
+	{ "TOGGLE_MUSIC", _s("Toggle music"), "C+m", nullptr },
+	{ "TOGGLE_SFX", _s("Toggle sfx"), "C+s", nullptr },
+	{ "TOGGLE_ORIGINAL_STYLE_COMMAND_BAR", _s("Show / hide original style command bar"), nullptr, nullptr },
+	{ "DO_ACTION", _s("Do action"), "RETURN", "KP_ENTER" },
+	{ "CANCEL_ACTION", _s("Cancel action"), "SPACE", nullptr },
+	{ "MSG_SCROLL_UP", _s("Msg scroll up"), "PAGEUP", nullptr },
+	{ "MSG_SCROLL_DOWN", _s("Msg scroll down"), "PAGEDOWN", nullptr },
+	{ "SHOW_KEYS", _s("Show keys"), "h", nullptr },
+	{ "DECREASE_DEBUG", _s("Decrease debug"), "C+d", nullptr },
+	{ "INCREASE_DEBUG", _s("Increase debug"), "C+i", nullptr },
+	{ "CLOSE_GUMPS", _s("Close gumps"), "z", nullptr },
+	//{ "USE_ITEM", _s("Use item"), nullptr, nullptr }, // TODO: this takes an item no parameter
+	{ "TOGGLE_ALT_CODE_MODE", _s("Enter ALT code (hold)"), "LALT", nullptr },
+	{ "ALT_CODE_0", _s("ALT code 0"), "A+KP0", "A+0" },
+	{ "ALT_CODE_1", _s("ALT code 1"), "A+KP1", "A+1" },
+	{ "ALT_CODE_2", _s("ALT code 2"), "A+KP2", "A+2" },
+	{ "ALT_CODE_3", _s("ALT code 3"), "A+KP3", "A+3" },
+	{ "ALT_CODE_4", _s("ALT code 4"), "A+KP4", "A+4" },
+	{ "ALT_CODE_5", _s("ALT code 5"), "A+KP5", "A+5" },
+	{ "ALT_CODE_6", _s("ALT code 6"), "A+KP6", "A+6" },
+	{ "ALT_CODE_7", _s("ALT code 7"), "A+KP7", "A+7" },
+	{ "ALT_CODE_8", _s("ALT code 8"), "A+KP8", "A+8" },
+	{ "ALT_CODE_9", _s("ALT code 9"), "A+KP9", "A+9" },
 };
 
 static const NuvieActionDescription CheatKeyDescriptions[] = {
-	{ "ASSET_VIEWER", "Open the asset viewer", nullptr, nullptr },
-	{ "SHOW_EGGS", "Show eggs", "e", nullptr },
-	{ "TOGGLE_HACKMOVE", "Toggle hack move", "A+h", nullptr },
-	{ "TOGGLE_EGG_SPAWN", "Toggle egg spawn", "A+C+e", nullptr },
-	{ "TOGGLE_NO_DARKNESS", "Toggle no darkness", "A+i", nullptr },
-	{ "TOGGLE_PICKPOCKET_MODE", "Toggle pickpocket mode", "A+C+p", nullptr },
-	{ "TOGGLE_GOD_MODE", "Toggle god mode", "A+C+g", nullptr },
-	{ "TOGGLE_ETHEREAL", "Toggle ethereal mode", "A+e", nullptr},
-	{ "TOGGLE_X_RAY", "Toggle X-ray mode", "x", nullptr },
-	{ "HEAL_PARTY", "Heal party", "A+C+h", nullptr },
-	{ "TELEPORT_TO_CURSOR", "Teleport to cursor", "C+t", nullptr },
-	{ "TOGGLE_CHEATS", "Toggle cheats", "C+c", nullptr },
+	{ "ASSET_VIEWER", _s("Open the asset viewer"), nullptr, nullptr },
+	{ "SHOW_EGGS", _s("Show eggs"), "e", nullptr },
```

Comment:
Perhaps eggs should be described for translations with an I18N comment.
Eggs in this context are invisible objects on the map that trigger game logic when "hatched".
Maybe someone else could describe that in better terms.

Reply:
What about
`// I18N: Eggs are in-game hatchable objects that are normally invisible`